blackduck/Client.py

```diff
@@ -252,3 +257,21 @@ def get_items(self, url, page_size=100, name='', **kwargs):
                 break
 
             offset += page_size
+
+    @staticmethod
+    def log_http_error(r):
+        """Log http error response
+
+        Args:
+            r (requests.HTTPError OR requests.Response): to log
+        """
+        if isinstance(r, requests.HTTPError):
+            r = r.response
+        logging.error(f"{r.request.method} {r.url}")
+        status_description = requests.status_codes._codes[r.status_code][0]
+        logging.error(f"HTTP response status code {r.status_code}: {status_description}")
+        try:
+            content = json.dumps(r.json(), indent=4)
+            logging.error(f"HTTP response json (formatted): {content}")
+        except json.JSONDecodeError:
+            logging.error(f"HTTP response text: {r.text}")
```

Comment:
This prints out everything a user will need to fix up their code to successfully GET or to print out something useful in the generic unhandled error during POST.

Reply:
If desired I can rename this method to handle_error() with the intention of the user to completely override it with their own implementation.